client/app/views/admin/network/tor.html

```diff
@@ -4,10 +4,17 @@
       <div class="row">
         <div class="col-md-12">
           <div class="form-group">
-              <label><span data-translate>Tor Onion Service</span>:
-                  <span ng-hide="admin.node.onionservice === ''"  class="label label-primary">{{admin.node.onionservice}}</span>
-                  <span ng-show="admin.node.onionservice === ''" class="label label-warning" data-translate>Not running</span>
-              </label>
+            <div ng-hide="admin.node.onionservice === ''">
```

Comment:
missing 'data-'

in addition we may use a simple data-ng-show="admin.node.onionservice"

no need for === ' '  on empty strings, we can use